.babelrc

```diff
@@ -1,89 +1,21 @@
-{ 
-  "plugins": [
-    "transform-es5-property-mutators",
-    "transform-es3-member-expression-literals",
-    "transform-es3-property-literals"
+{
+  "presets": [
+    "es2015-rollup"
```

Comment:
What does `es2015-rollup` have that `shopify/web` and `shopify/node` don't? If there's something crucial it would be good to review adding to `shopify`